app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -114,7 +114,7 @@ $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium',
                 </tr>
                 <tr>
                     <td class="label"><label><?php echo Mage::helper('sales')->__('Email') ?></label></td>
-                    <td class="value"><a href="mailto:<?php echo $this->escapeHtml($_order->getCustomerEmail()) ?>"><strong><?php echo $this->escapeHtml($_order->getCustomerEmail()) ?></strong></a></td>
+                    <td class="value"><a href="mailto:<?php echo $this->escapeHtml($_order->getCustomerEmail()) ?>" class="copy-text" data-text="<?= $_order->getCustomerEmail()?>" data-copy-icon='{"cursor":"pointer","margin":"2px 0 0 2px","height":"10px","width":"10px"}'><strong><?php echo $this->escapeHtml($_order->getCustomerEmail()) ?></strong></a></td>
```

Comment:
all short syntax `<?=` must be `<?php echo `
There was a proposal, but it didn't succeed https://github.com/OpenMage/magento-lts/pull/3936

Reply:
fixed it

Reply:
It's fine to use old echo format.